examples/resources/customer/main.tf

```diff
@@ -54,10 +54,10 @@ provider "vmp" {
 ##########################################
```

Comment:
I may be mistaken, but I think the folder name has to match the actual resource name e.g. "vmp_customer". This is something the documentation tool expects.